pkg/worker/handler/endpoint/handler.go

```diff
@@ -0,0 +1,95 @@
+package endpoint
+
+import (
+	"fmt"
+
+	"github.com/0xSplits/specta/pkg/envvar"
+	"github.com/0xSplits/specta/pkg/recorder"
+	"github.com/0xSplits/specta/pkg/registry"
+	"github.com/xh3b4sd/logger"
+	"github.com/xh3b4sd/tracer"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const (
+	Metric = "http_endpoint_health"
+)
+
+var (
+	endpoint = map[string]map[string]string{
+		"server": {
+			"testing":    "https://test.api.splits.org/metrics",
+			"staging":    "https://beta.api.splits.org/metrics",
+			"production": "https://api.splits.org/metrics",
+		},
```

Comment:
@mihoward21 we need to set the new DNS here for the server once we fixed those endpoints according to the new hosted zones.